README.md

```diff
@@ -34,35 +34,57 @@ You can either [clone the repository from GitHub](https://github.com/jtebert/kil
 
 ### Dependencies
 
-[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with: `sudo apt install libsfml-dev libhdf5-dev libomp-dev`
+[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with:
 
-[nlohmann/json](https://github.com/nlohmann/json): Add [`json.hpp`](https://github.com/nlohmann/json/releases/download/v3.5.0/json.hpp) to the `include/` folder. **Note:** This links to the most recent release (3.5) that has been verified for compatibility. You can find other [versions and release notes on Github](https://github.com/nlohmann/json/releases).
+    sudo apt install libsfml-dev libhdf5-dev libomp-dev
 
 ### Compile
 
-Create a static library at `bin/libKilosim.a`:
+To compile, copy and paste the following:
 
-    make
-or
+    mkdir build                                #Create a build directory
+    cd build                                   #Move into build directory
+    cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .. #Configure for your system
+    make                                       #Build all
+    make install                               #Install files
```

Comment:
I'm getting the following error here:

```
[100%] Built target kilosim
Install the project...
-- Install configuration: "RelWithDebInfo"
CMake Error at cmake_install.cmake:55 (file):
  file INSTALL cannot find
  "/home/jtebert/google-drive/Harvard/SSR/kilobots/kilosim-richard-make/build/kilosim_example".

Makefile:85: recipe for target 'install' failed
make: *** [install] Error 1
```

This doesn't occur if I first do `make examples`, which generates an executable in the build directory. I'm also unclear on what each of these make commands is doing (like what the difference is between `make`, `make install`, and `make kilosim`)

For example, `make examples` creating an executable in the `build` directory, but `make install` will put it in the `bin` directory (but only if `build examples` has already been run?)

Reply:
I've made the `make examples` install optional, which should fix this.